packages/eslint-plugin-js/package.json

```diff
@@ -100,11 +100,17 @@
     "dts",
     "rules/**/*.d.ts"
   ],
+  "engines": {
+    "node": "^16.0.0 || >=18.0.0"
+  },
   "scripts": {
     "build": "rimraf dist && rollup --config=rollup.config.mts --configPlugin=rollup-plugin-esbuild",
     "dev": "rollup --config=rollup.config.mts --configPlugin=rollup-plugin-esbuild --watch",
     "prepublishOnly": "pnpm build"
   },
+  "peerDependencies": {
+    "eslint": ">=8.0.0"
```

Comment:
I think we probably need the version at the time we forked it. I don't think it will work unless we use at least a newer version than v8.40.

https://github.com/eslint-stylistic/eslint-stylistic/issues/45#issuecomment-1793911075